cmd/parca-agent/main.go

```diff
@@ -285,6 +285,7 @@ func run(logger log.Logger, reg *prometheus.Registry, flags flags) error {
 				metadata.Target(flags.Node, flags.MetadataExternalLabels),
 				metadata.Cgroup(),
 				metadata.Compiler(),
+				metadata.System(),
```

Comment:
You may want to remove changes that were introduced in this PR from your branch: https://github.com/parca-dev/parca-agent/pull/858? Not sure if it's some weird Github thing though.

Reply:
Is this a github glitch? I don't see these changes? 🤔